client/components/button/docs/example.jsx

```diff
@@ -59,6 +59,12 @@ var Buttons = React.createClass( {
 						<Button primary disabled >Primary disabled button</Button>
 					</div>
 					<div className="design-assets__button-row">
+						<Button secondary >Secondary button</Button>
```

Comment:
(To be clear: if it's a self-closing tag, then it is expected to keep a space: `<Button />`)